features/step_definitions/request_of_delivery.rb

```diff
@@ -0,0 +1,11 @@
+Given(/^"([^"]*)" has selected "([^"]*)" button on the "([^"]*)"$/) do |arg1, arg2, arg3|
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+When(/^I click on "([^"]*)" for order "([^"]*)"$/) do |arg1, arg2|
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+When(/^I enter "([^"]*)" in "([^"]*)"$/) do |arg1, arg2|
+  pending # Write code here that turns the phrase above into concrete actions
+end
```

Comment:
Hmm... And this definitions? Not used? I would remove them.

Reply:
not used. Left over from previous testing

Reply:
i was created when we I started to work on the feature